provider/oci/instance.go

```diff
@@ -176,8 +178,8 @@ func (o *ociInstance) waitForPublicIP(ctx envcontext.ProviderCallContext) error
 		if err != nil {
 			return err
 		}
-		if iteration >= 30 {
-			logger.Warningf("Instance still in running state after %v checks. breaking loop", iteration)
+		if iteration >= maxPollIterations {
+			logger.Debugf("Still no public IP after %v checks. breaking loop", iteration)
```

Comment:
maybe this would be clearer if we used a
startTime := time.Now()
before entering the loop, and then

logger.Debugf("could not find a public IP after %s", time.Since(startTime))